### PR TITLE
more detail display for morale and luck

### PR DESCRIPTION
**Pull request by undef21**
related issue #2808

### CODE REVIEW

File: src/fheroes2/heroes/heroes_indicator.h [hunk 1]
```diff
@@ -29,8 +29,8 @@
 
 namespace fheroes2
 {
-    const char * LuckString( const int luck );
-    const char * MoraleString( const int morale );
+    const std::string LuckString( const int luck );
+    const std::string MoraleString( const int morale );
```
Comment (ihhub) [inline src/fheroes2/heroes/heroes_indicator.h:33]: Please remove `const` modifier from return value.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @undef21 , I left just one comment. Please take a look.

File: src/fheroes2/heroes/heroes_indicator.h (hunk 1 above)
Comment (undef21) [reply src/fheroes2/heroes/heroes_indicator.h]: Done.

**Comment by ihhub** [comment]
@undef21 , a bunch of thanks for fixing this issue!

**Pull request merged**